public/app/features/search/service/sql.ts

```diff
@@ -225,8 +225,59 @@ export class SQLSearcher implements GrafanaSearcher {
     };
   }
 
+  async getFolderChildren(parentUid?: string): Promise<NestedFolderItem[]> {
```

Comment:
This could be outside the GrafanaSearcher functions and instead just call `getGrafanaSearcher()` for the search it needs to make